sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,203 @@
+# SIP-028: sBTC Signer Criteria
+
+**SIP Number:** 028  
+**Title:** Signer Criteria for sBTC, A Decentralized and Programmable Asset Backed 1:1 with BTC  
+**Consideration:** Technical, Economics  
```

Comment:
```suggestion
**Consideration:** Technical, Economic  
```
According to the SIP Specification, the Consideration should be 'Economic'. Please correct 'Economics' to 'Economic'.